### PR TITLE
Day eleven

### PR DESCRIPTION
**Pull request by Qairkhan**
Использована деструктуризация в компоненте profile. Добавлены reduce для делегирования метода dispatch. (форматирование по prettier  будет сделано в ближайшее время)

### CODE REVIEW

File: src/components/Dialogs/DialogItem/DialogItem.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+import { ROUTES, I18N } from "../../../core/constants";
+
+import s from "./DialogItem.module.css"
+
+const DialogItem = (props) => {
+  const way = `${ROUTES.MESSAGES}/${props.id}`;
```
Comment (Dmitry-White) [inline src/components/Dialogs/DialogItem/DialogItem.jsx]: Consider using destructuring of "props" for "id" and "name" in this function.

File: src/redux/dialogsReducer.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import ACTION_TYPES from "./actionTypes";
+
+const dialogsReducer = (state, action) => {
+    switch(action.type) {
+        case ACTION_TYPES.UPDATE_NEW_MESSAGE_BODY: 
+        state.newMessageBody = action.payload;
+        return state;
+
+case ACTION_TYPES.SEND_MESSAGE:
```
Comment (Dmitry-White) [inline src/redux/dialogsReducer.js]: There's something wrong with indentation - it's hard to read.

File: src/redux/dialogsReducer.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import ACTION_TYPES from "./actionTypes";
+
+const dialogsReducer = (state, action) => {
+    switch(action.type) {
+        case ACTION_TYPES.UPDATE_NEW_MESSAGE_BODY: 
+        state.newMessageBody = action.payload;
```
Comment (Dmitry-White) [inline src/redux/dialogsReducer.js]: There's something wrong with indentation.

File: src/redux/dialogsReducer.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        return state;
+
+case ACTION_TYPES.SEND_MESSAGE: 
+
+    let body = state.newMessageBody;
```
Comment (Dmitry-White) [inline src/redux/dialogsReducer.js]: Consider using "const" instead of "let"

File: src/redux/profileReducer.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    switch(action.type) {
+        case ACTION_TYPES.ADD_POST: 
+        const newPost = {
+            id: 5,
+            message: state.newPostText,
+    
+            likesCount: 0,
+          };
+    
+          state.postsData.push(newPost);
```
Comment (Dmitry-White) [inline src/redux/profileReducer.js]: There's something wrong with indentation.

**Review by Dmitry-White — CHANGES_REQUESTED** [review]

File: src/redux/dialogsReducer.js (hunk 4 above)
Comment (Qairkhan) [reply src/redux/dialogsReducer.js]: done

File: src/redux/profileReducer.js (hunk 5 above)
Comment (Qairkhan) [reply src/redux/profileReducer.js]: done

File: src/redux/dialogsReducer.js (hunk 3 above)
Comment (Qairkhan) [reply src/redux/dialogsReducer.js]: done

File: src/redux/dialogsReducer.js (hunk 2 above)
Comment (Qairkhan) [reply src/redux/dialogsReducer.js]: done

File: src/components/Dialogs/DialogItem/DialogItem.jsx (hunk 1 above)
Comment (Qairkhan) [reply src/components/Dialogs/DialogItem/DialogItem.jsx]: оставлен комментарий , будет реализовано в будущем

**Review by Dmitry-White — APPROVED** [review]
Nice work!
Now it definitely looks much better!

**Pull request merged**